go.mod

```diff
@@ -3,7 +3,7 @@ module github.com/ktrysmt/go-bitbucket
 go 1.12
 
 require (
-	github.com/golang/protobuf v1.0.0
+	github.com/golang/protobuf v1.0.0 // indirect
```

Comment:
Would you remove the line if it is not a problem?

Reply:
Got it